client/src/views/files/FoldersPage.vue

```diff
@@ -619,10 +633,11 @@ async function openEntries(entries: parsec.EntryStat[]): Promise<void> {
   if (entries.length !== 1) {
     return;
   }
-  if (parsec.isWeb()) {
+  if (!parsec.isWeb()) {
```

Comment:
Nop, should be `if (parsec.isWeb())`

Reply:
It was just for a preview, a simple oversight.